### PR TITLE
Trivia: Display question numbers

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

File: server/chat-plugins/trivia.ts [hunk 1]
```diff
@@ -612,7 +612,7 @@ class Trivia extends Rooms.RoomGame {
 	 */
 	sendQuestion(question: TriviaQuestion) {
 		this.broadcast(
-			`Question: ${question.question}`,
+			`Question ${this.questionNumber}: ${question.question}`,
```
Comment (KrisXV) [inline server/chat-plugins/trivia.ts:615]:
```suggestion
			`Question #${this.questionNumber}: ${question.question}`,
```

Comment (AnnikaCodes) [reply server/chat-plugins/trivia.ts:615]: Personally, I think Question 1, Question 2, etc, is cleaner. I don't really care either way, though.

Comment (KrisXV) [reply server/chat-plugins/trivia.ts:615]: That's fair, i also dont really care either way

Comment (Zarel) [reply server/chat-plugins/trivia.ts:615]: On the other hand there's an argument for preferring "#1" over "Question 1".

**Pull request merged**